lib/ObisNames.js

```diff
@@ -6,352 +6,352 @@
+            49: "L2 Scheinleistung Bezug +",
+            50: "L2 Scheinleistung Abgabe -",
+            51: "L2 Strom",
+            52: "L2 Spannung",
+            53: "L2 Leistungsfaktor",
+            61: "L3 Wirkarbeit Bezug +",
+            62: "L3 Wirkarbeit Abgabe -",
+            63: "L3 Blindarbeit Bezug +",
+            64: "L3 Blindarbeit Abgabe -",
+            65: "L3 Blindarbeit Q I",
+            66: "L3 Blindarbeit Q II",
+            67: "L3 Blindarbeit Q III",
+            68: "L3 Blindarbeit Q IV",
+            69: "L3 Scheinleistung Bezug +",
+            70: "L3 Scheinleistung Abgabe -",
+            71: "L3 Strom",
+            72: "L3 Spannung",
+            73: "L3 Leistungsfaktor",
+            81: "Winkel",
+            82: "Anzahl ohne Einheit",
+            91: "Neutraler Strom",
+            92: "Neutrale Spannung",
+            94: "Länderspezifisch",
+            96: "Wartungseintrag",
+            97: "Fehler",
+            98: "Listen",
+            99: "Datenprofile, Lastgang P.01/ P.02, Betriebslogbuch P.98/ P.99"
         },
         "nl": {
             0: "Algemeen doel",
-            1: "Totaal aankoop actieve energie +",
-            2: "Totaal afgegeven actieve energie -",
-            3: "Totale aankoop reactieve energie +",
-            4: "Totale reactieve energie-output -",
-            5: "Totale reactieve energie Q I",
-            6: "Totale reactieve energie Q II",
-            7: "Totale reactieve energie Q III",
-            8: "Totale reactieve energie Q IV",
-            9: "Som schijnbare stroomafname +",
-            10: "Totaal schijnbaar vermogen -",
-            11: "Actueel",
+            1: "Totale werklastenergie afname +",
+            2: "Totale werklastenergie teruggave -",
+            3: "Totale blindlastenergie afname +",
+            4: "Totale blindlastenergie teruggave -",
+            5: "Totale blindlastenergie Q I",
+            6: "Totale blindlastenergie Q II",
+            7: "Totale blindlastenergie Q III",
+            8: "Totale blindlastenergie Q IV",
+            9: "Totale schijnbelasting afname +",
+            10: "Totale schijnbelasting teruggave -",
+            11: "Stroom",
             12: 'Voltage',
-            13: "Vermogensfactor",
-            14: 'Frequency',
-            16: "Totaal actief vermogen",
-            21: "L1 referentie actieve energie +",
-            22: "L1 uitgang van actieve energie -",
-            23: "L1-referentie reactieve energie +",
-            24: "L1 uitgang reactieve energie -",
-            25: "L1 reactieve energie Q I",
-            26: "L1 reactieve energie Q II",
-            27: "L1 reactieve energie Q III",
-            28: "L1 Reactieve energie Q IV",
-            29: "L1 schijnbaar vermogen referentie +",
-            30: "L1 schijnbaar vermogen -",
+            13: "Prestatiefactor",
+            14: 'Frequentie',
+            16: "Totaalprestatie",
+            21: "L1 werklastenergie afname +",
```

Comment:
same here, werklast ? Ik heb het gevoel da de EN beschrijvingen hier al erg vaag zijn ?

Reply:
Ja, ik weet ook niet wat hiermee bedoeld worden. Het is de officiële vertaling, maar ik denk dat de gemiddelde Io-Broker gebruiker hier niets mee kan.